src/composables/RoutingSlip/useCreateRoutingSlip.ts

```diff
@@ -8,6 +13,11 @@ export function useCreateRoutingSlip (_, context) {
   const createRoutingSlipPaymentRef = ref<HTMLFormElement>()
   const modalDialogRef = ref<HTMLFormElement>()
 
+  // vuex action and state
+  const { createRoutingSlip } = useActions(['createRoutingSlip'])
+  const { resetRoutingSlipDetails } = useActions(['resetRoutingSlipDetails'])
```

Comment:
this both line can be combined const { createRoutingSlip, resetRoutingSlipDetails } = useActions(['createRoutingSlip', 'resetRoutingSlipDetails'])

Reply:
thanks :)